### ISSUE TITLE
Support for Ubuntu

### ISSUE THREAD

**Issue by thanikkal** [uptimejp/postgres-toolkit#7]
The supported operating system list doesnt have Ubuntu, are there any plans to support ubuntu?

**Comment by snaga** [comment]
I'm not familiar with Ubuntu so far, but I guess the toolkit can work if you have "python2.6" command in your path. This toolkit is mostly written in python, so it's worth a try if you have some Ubuntu box. :) And please let me know if something goes wrong.

**Comment by ThomasG77** [comment]
It works but it needs some dirty hacks.

You have to do :
```bash
wget http://dl.uptimeforce.com/postgres-toolkit/install.sh`
chmod +x install.sh
```
Because installation is done in /opt, you need `sudo` rights
```bash
sudo ./install.sh
```
Then, you add to the `PATH`, the bin folder using:

`export PATH=$PATH:/opt/uptime/postgres-toolkit-0.2/bin`

It should be added in your `~/.bashrc`

When you try to run `pt-config`, it complains because there is an explicit call to python2.6 in each executable file e.g [this example](https://github.com/uptimejp/postgres-toolkit/blob/develop/bin/pt-config#L1)
Nowadays, Python version bundled in Ubuntu is version 2.7

Hence, you have to make a symbolic link and you can run the executables.
`sudo ln -s /usr/bin/python2.7 /usr/bin/python2.6`

This last step could be removed IMO. According to Python docs, it should be enough to set `#!/usr/bin/env python` instead of `#!/usr/bin/env python2.6` https://docs.python.org/2.7/using/unix.html#miscellaneous, It's always possible to catch Python version using `import sys;print sys.version_info`

**Comment by snaga** [comment]
Thanks for such a precise report!
I will fix them in the develop branch (and the next release) to allow ubuntu asap.
Please wait a moment.

**Comment by snaga** [comment]
Created an experimental branch for Ubuntu, and tested with Ubuntu 14.04.2 and PostgreSQL 9.3.
https://github.com/uptimejp/postgres-toolkit/tree/fix/ubuntu
aa5015c1a5b797b8410fbe35c7e666e9cfe1705e

It will be merged into the develop branch asap after passing regression tests with other PostgreSQL major versions (9.0, 9.1, 9.2 and 9.4).

**Comment by snaga** [comment]
Hi,

I just released 0.2.1, which supports RHEL 7/CentOS 7/Ubuntu 14.04 LTS with Python 2.7.
Please check it out if you are using those distros.

http://postgres-toolkit.readthedocs.org/en/latest/install.html
http://postgres-toolkit.readthedocs.org/en/latest/releasenote.html

Regards,

**Comment by snaga** [comment]
Postgres Toolkit 0.2.1 released
http://www.postgresql.org/message-id/CAA8sozc0cYX9v9v62VJ6drn+FnD6jNNF8kSozveP5S5oU20KZg@mail.gmail.com
